### DISCUSSION TITLE
No encuentro el  video de  api usarios

### DISCUSSION THREAD

**Discussion by jmlimas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#514] [General]
Hola, sigo insistiendo que  la  organización de los  videos es un revoltijo, me  pueden  decir  donde  encuentro el  video  de los  usuarios  donde  se  uso una  api,  no logro  encontrarlo.

Saludos

**Comment by JLuisPrz** [comment]
EL código de generar usuarios random esta [aquí](https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/05%20-%20VUEJS/programas/CDN), el video esta en teams.

**Reply by jmlimas** [reply]
Muchas  Gracias JLuisPrz..!!
